packages/composites/vesper/test/tvl.test.ts

```diff
@@ -0,0 +1,12 @@
+import { getTokenAllocations } from '../src/tvl'
```

Comment:
Since this is an e2e test, could we nest this test one deeper in an `e2e` folder?